.github/workflows/integration-test.yml

```diff
@@ -54,11 +54,31 @@ jobs:
           name: agent-bundle-linux-${{ matrix.ARCH }}
           path: ./dist/agent-bundle_linux_${{ matrix.ARCH }}.tar.gz
 
+  otelcol:
+    runs-on: ubuntu-20.04
```

Comment:
Is it possible to use a more recent ubuntu?

Reply:
I think so.  Our compile jobs (which this job is based on) currently run on `ubuntu-20.04`, for example https://github.com/signalfx/splunk-otel-collector/blob/main/.github/workflows/build-and-test.yml#L214.  So we can try updating them to a newer runner in a separate PR.